app/templates/buyers/_requirements_meta.html

```diff
@@ -0,0 +1,13 @@
+<div id="requirements-meta">
+  <h2 class="sidebar-heading">Published</h2>
+  <p class="sidebar-content">{{ brief.publishedAt|dateformat }}</p>
+  {% if brief.status == 'closed' %}
+    <h2 class="sidebar-heading">Closed</h2>
+    <p class="sidebar-content">{{ brief.applicationsClosedAt|dateformat }}</p>
+  {% else %}
+    <h2 class="sidebar-heading">Closing</h2>
+    <p class="sidebar-content">{{ brief.applicationsClosedAt|dateformat }}</p>
+  {% endif %}
+  <h2 class="sidebar-heading">Framework</h2>
+  <p class="framework-name"><a href="{{ framework_agreement_url }}">{{ brief.frameworkName }}</a></p>
```

Comment:
I'm gonna go keep. It means if we wanted to do something different with the framework name in future it makes it easier.